Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -333,48 +328,63 @@ void Tileset::_updateLodTransitions(
 
 const ViewUpdateResult&
 Tileset::updateViewOffline(const std::vector<ViewState>& frustums) {
```

Comment:
I never learned the nuances behind `updateViewOffline`, but does this imply there should be an `updateViewGroupOffline` too?